pubsub/spec/sample_spec.rb

```diff
@@ -0,0 +1,146 @@
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../sample"
+require "spec_helper"
+
+describe "Pub/Sub sample" do
+  TOPIC_NAME = "my-topic".freeze
```

Comment:
Rubocop again, it want's globals to be immutable. I'll remove the `.freeze` and update the rubocop config.